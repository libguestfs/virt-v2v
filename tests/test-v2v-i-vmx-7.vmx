.encoding = "UTF-8"
config.version = "8"
virtualHW.version = "17"
vmci0.present = "TRUE"
floppy0.present = "FALSE"
numvcpus = "2"
memSize = "4096"
bios.bootRetry.delay = "10"
firmware = "efi"
powerType.suspend = "soft"
tools.upgrade.policy = "manual"
sched.cpu.units = "mhz"
sched.cpu.affinity = "all"
vm.createDate = "1649413453022013"
scsi0.virtualDev = "lsisas1068"
scsi0.present = "TRUE"
sata0.present = "TRUE"
usb_xhci.present = "TRUE"
scsi0:0.deviceType = "scsi-hardDisk"
scsi0:0.fileName = "win2019.vmdk"
sched.scsi0:0.shares = "normal"
sched.scsi0:0.throughputCap = "off"
scsi0:0.present = "TRUE"
ethernet0.virtualDev = "vmxnet3"
ethernet0.networkName = "VM Network"
ethernet0.addressType = "generated"
ethernet0.wakeOnPcktRcv = "FALSE"
ethernet0.present = "TRUE"
sata0:0.deviceType = "cdrom-image"
sata0:0.fileName = "/vmfs/volumes/624d796f-52b87554-ccba-a4ae111c9b1b/iso/en-us_windows_server_2019_updated_aug_2021_x64_dvd_a6431a28.iso"
sata0:0.present = "TRUE"
displayName = "win2019"
guestOS = "windows2019srv-64"
uefi.secureBoot.enabled = "TRUE"
toolScripts.afterPowerOn = "TRUE"
toolScripts.afterResume = "TRUE"
toolScripts.beforeSuspend = "TRUE"
toolScripts.beforePowerOff = "TRUE"
tools.syncTime = "FALSE"
uuid.bios = "56 4d 6c 27 96 ea 12 07-6f 0b b1 1d 81 46 9b 0b"
uuid.location = "56 4d 6c 27 96 ea 12 07-6f 0b b1 1d 81 46 9b 0b"
vc.uuid = "52 bb 00 73 da d0 a0 c7-ad 14 03 81 13 03 ed fe"
sched.cpu.min = "0"
sched.cpu.shares = "normal"
sched.mem.min = "0"
sched.mem.minSize = "0"
sched.mem.shares = "normal"
ethernet0.generatedAddress = "00:0c:29:46:9b:0b"
vmci0.id = "-2126079221"
cleanShutdown = "TRUE"
sata0:0.startConnected = "TRUE"
extendedConfigFile = "win2019.vmxf"
ethernet0.uptCompatibility = "TRUE"
sata0:3.fileName = "win2019_1.vmdk"
sched.sata0:3.shares = "normal"
sched.sata0:3.throughputCap = "off"
sata0:3.present = "TRUE"
tools.guest.desktop.autolock = "FALSE"
nvram = "win2019.nvram"
pciBridge0.present = "TRUE"
svga.present = "TRUE"
pciBridge4.present = "TRUE"
pciBridge4.virtualDev = "pcieRootPort"
pciBridge4.functions = "8"
pciBridge5.present = "TRUE"
pciBridge5.virtualDev = "pcieRootPort"
pciBridge5.functions = "8"
pciBridge6.present = "TRUE"
pciBridge6.virtualDev = "pcieRootPort"
pciBridge6.functions = "8"
pciBridge7.present = "TRUE"
pciBridge7.virtualDev = "pcieRootPort"
pciBridge7.functions = "8"
hpet0.present = "TRUE"
RemoteDisplay.maxConnections = "-1"
sched.cpu.latencySensitivity = "normal"
svga.autodetect = "TRUE"
disk.EnableUUID = "TRUE"
numa.autosize.cookie = "20012"
numa.autosize.vcpu.maxPerVirtualNode = "2"
sched.swap.derivedName = "/vmfs/volumes/624d796f-52b87554-ccba-a4ae111c9b1b/win2019/win2019-645c6913.vswp"
vm.genidX = "-5104727847056752683"
pciBridge0.pciSlotNumber = "17"
pciBridge4.pciSlotNumber = "21"
pciBridge5.pciSlotNumber = "22"
pciBridge6.pciSlotNumber = "23"
pciBridge7.pciSlotNumber = "24"
scsi0.pciSlotNumber = "160"
ethernet0.pciSlotNumber = "192"
usb_xhci.pciSlotNumber = "224"
vmci0.pciSlotNumber = "32"
sata0.pciSlotNumber = "33"
scsi0.sasWWID = "50 05 05 67 96 ea 12 00"
vmotion.checkpointFBSize = "4194304"
vmotion.checkpointSVGAPrimarySize = "16777216"
vmotion.svga.mobMaxSize = "16777216"
vmotion.svga.graphicsMemoryKB = "16384"
ethernet0.generatedAddressOffset = "0"
monitor.phys_bits_used = "45"
softPowerOff = "TRUE"
toolsInstallManager.lastInstallError = "0"
toolsInstallManager.updateCounter = "3"
svga.guestBackedPrimaryAware = "TRUE"
tools.remindInstall = "FALSE"
vm.genid = "-802103094701856339"
usb_xhci:4.present = "TRUE"
usb_xhci:4.deviceType = "hid"
usb_xhci:4.port = "4"
usb_xhci:4.parent = "-1"
migrate.hostLog = "./win2019-645c6913.hlog"
