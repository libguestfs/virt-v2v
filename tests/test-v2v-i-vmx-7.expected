
Source guest information (--print-source option):

    source name: win2019
hypervisor type: vmware
       VM genid: 2f0de9ad-5b7c-f4de-d53f-23b1115d28b9
         memory: 4294967296 (bytes)
       nr vCPUs: 2
     CPU vendor: 
      CPU model: 
   CPU topology: 
   CPU features: 
       firmware: uefi
        display: 
          sound: 
disks:
	0 [scsi]
	1 [sata]
removable media:

NICs:
	Network "VM Network" mac: 00:0c:29:46:9b:0b [vmxnet3]

